WcaOnRails/app/views/competitions/_nearby_competitions.html.erb

```diff
@@ -10,14 +10,12 @@
   </label>
   <div class="col-sm-9">
     <% if @competition.has_date? && @competition.has_location? %>
-      <% nearby_competitions = @competition.nearby_competitions(Competition::NEARBY_DAYS_WARNING, Competition::NEARBY_DISTANCE_KM_WARNING)[0, 10] %>
-      <% if nearby_competitions.empty? %>
+      <% if @nearby_competitions.empty? %>
         <%= alert :success, t('.no_comp_nearby') %>
       <% else %>
-        <%= render 'nearby_competitions_table', nearby_competitions: nearby_competitions %>
+        <%= render 'nearby_competitions_table', nearby_competitions: @nearby_competitions %>
       <% end %>
     <% else %>
-      <% nearby_competitions = [] %>
```

Comment:
It was hear so line 61 doesn't crash trying to loop over a nil pointer =) See https://travis-ci.org/thewca/worldcubeassociation.org/builds/172613292#L1401.